src/pages/ProjectDetail.tsx

```diff
@@ -0,0 +1,136 @@
+// api 연결시: import { useParams } from "react-router-dom"
+import styled from "styled-components";
+import { projectDetail, urlItem } from "../mocks/project";
+import { useState } from "react";
+
+const Container = styled.div`
+  background-color: #000;
+  color: white;
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+  padding: 2rem;
+`;
+
+const Row = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  gap: 2rem;
+`;
+
+const Label = styled.a`
+  background: white;
+  color: black;
+  padding: 0.5rem 1rem;
+  border-radius: 0.5rem;
+  font-size: 0.875rem;
+`;
+
+const Status = styled.span`
+  font-size: 1rem;
+  padding: 0.5rem 0.75rem;
+  border-radius: 0.375rem;
+  background: #dfffa3;
+  color: #111;
+  font-weight: bold;
+`;
+
+const ImageWrapper = styled.div`
+  position: relative;
+  width: 100%;
+  max-width: 600px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+const Image = styled.img`
+  width: auto;
+  height: 50rem;
+  border-radius: 1rem;
+`;
+
+const ArrowButton = styled.button<{ left?: boolean }>`
+  position: absolute;
+  top: 50%;
+  ${(props) => (props.left ? "left: 0.5rem;" : "right: 0.5rem;")}
+  transform: translateY(-50%);
+  background: rgba(255, 255, 255, 0.2);
+  border: none;
+  border-radius: 50%;
+  padding: 0.25rem;
+  cursor: pointer;
+  color: white;
+
+  &:disabled {
+    opacity: 0.3;
+    cursor: not-allowed;
+  }
+`;
+
+export default function ProjectDetailPage() {
+  const [index, setIndex] = useState(0);
+  const project = projectDetail;
+
+  const getStatusLabel = () => {
+    if (!project.is_active)
+      return project.project_type === "SERVICE" ? "서비스 종료" : "활동 종료";
+    return project.project_type === "SERVICE" ? "서비스 중" : "활동 중";
+  };
+
+  const labelOrder = [
+    "Android",
+    "iOS",
+    "Web",
+    "Github: Android",
+    "Github: iOS",
+    "Github: Web",
+  ];
+
+  const sortedUrls = labelOrder
+    .map((title) => project.urls.find((url) => url.title === title))
+    .filter((url): url is urlItem => !!url);
```

Comment:
falsy한 값을 비교하는 것 보다 그냥 직접적으로 undefined인지 아닌지를 파악하는 것이 더 가독성 있을 거 같습니다.
또한 labelOrder에 UrlType을 달아두면 굳이 타입 가드 필요 없지 않을까요? 한번 해보시면 좋을 거 같아요. 뒤에 undefined 필터도 붙을 일이 있는지 잘 모르겠네여.
```suggestion
  const sortedUrls = labelOrder
    .map((title) => project.urls.find((url) => url.title === title))
    .filter((url) => url !==undefined);
```